menas/src/main/scala/za/co/absa/enceladus/menas/controllers/RestExceptionHandler.scala

```diff
@@ -67,10 +75,23 @@ class RestExceptionHandler {
   }
 
   @ExceptionHandler(Array(classOf[OozieActionException]))
-   def handleOozieActionException(ex: OozieActionException): ResponseEntity[Object] = {
+  def handleOozieActionException(ex: OozieActionException): ResponseEntity[Object] = {
     val err = RestError(ex.getMessage)
     logger.error(s"Exception: $err", ex)
     new ResponseEntity(err, HttpStatus.INTERNAL_SERVER_ERROR)
-   }
-  
+  }
+
+  @ExceptionHandler(Array(classOf[OozieClientException]))
+  def handleOozieClientException(ex: OozieClientException): ResponseEntity[Object] = {
```

Comment:
```suggestion
  def handleOozieClientException(ex: OozieClientException): ResponseEntity[RestError] = {
```
Seems like a better fit.